test/no-identical-title.js

```diff
@@ -29,6 +29,8 @@ test(() => {
 			header + 'test.beforeEach(t => {}); test.beforeEach(t => {});',
 			header + 'test.afterEach(t => {}); test.afterEach(t => {});',
 			header + 'test.cb.before(t => {}); test.before.cb(t => {});',
+			header + 'test(t => {}); test(t => {});',
```

Comment:
Maybe add a comment here - "multiple anonymous tests covered by the if-multiple rule"